<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>857</width>
    <height>189</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QFrame" name="frame_cancelacionEvento">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="Line" name="line">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(178, 121, 36);</string>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QFrame" name="base_horas_eventos">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>200</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <property name="leftMargin">
           <number>7</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>9</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QFrame" name="frame_3">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>250</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Fecha:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_fecha">
               <property name="text">
                <string>TextLabel</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Hora: </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_hora">
               <property name="text">
                <string>TextLabel</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="Line" name="line_4">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(178, 121, 36);</string>
            </property>
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_2">
            <property name="minimumSize">
             <size>
              <width>400</width>
              <height>0</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="label_titulo">
               <property name="font">
                <font>
                 <pointsize>22</pointsize>
                 <bold>false</bold>
                </font>
               </property>
               <property name="text">
                <string>Corona de oro</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_descripcion">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>100</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>100</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">.QLabel{
    font-size: 14px; /* Tamaño de fuente normal */
    color: #666; /* Color de texto más claro */
}</string>
               </property>
               <property name="text">
                <string>Exposicion de la corona de oro</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="Line" name="line_5">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(178, 121, 36);</string>
            </property>
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <property name="spacing">
              <number>0</number>
             </property>
             <item>
              <widget class="QPushButton" name="boton_CancelarEvento">
               <property name="minimumSize">
                <size>
                 <width>120</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>120</width>
                 <height>50</height>
                </size>
               </property>
               <property name="baseSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">/* Estilo para QPushButton */
.QPushButton {
    background-color: #c57007; /* Color de fondo */
    color: white; /* Color de texto */
    border: none; /* Sin borde */
    border-radius: 5px; /* Esquinas redondeadas */
    padding: 8px 16px; /* Espaciado interno */
    font-size: 14px; /* Tamaño de fuente */
    font-weight: bold; /* Fuente en negrita */
}

.QPushButton:hover {
     color: #fff; /* Color del texto al pasar el cursor */
    background-color:#f49011 ;/* Cambiar el color de fondo al pasar el mouse */
    border-color:#f49011;/* Cambia el color de fondo al pasar el mouse */
}

QPushButton:presset{
	 background-color: #c57007;
}
</string>
               </property>
               <property name="text">
                <string>Cancelar</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="Boton_rechazarCancelacion">
               <property name="minimumSize">
                <size>
                 <width>120</width>
                 <height>50</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>120</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">/* Estilo para QPushButton */
.QPushButton {
    background-color: #c57007; /* Color de fondo */
    color: white; /* Color de texto */
    border: none; /* Sin borde */
    border-radius: 5px; /* Esquinas redondeadas */
    padding: 8px 16px; /* Espaciado interno */
    font-size: 14px; /* Tamaño de fuente */
    font-weight: bold; /* Fuente en negrita */
}

.QPushButton:hover {
     color: #fff; /* Color del texto al pasar el cursor */
    background-color:#f49011 ;/* Cambiar el color de fondo al pasar el mouse */
    border-color:#f49011;/* Cambia el color de fondo al pasar el mouse */
}

QPushButton:presset{
	 background-color: #c57007;
}
</string>
               </property>
               <property name="text">
                <string>Rechazar</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
